src/NRedisStack/Search/Literals/FieldOptions.cs

```diff
@@ -1,11 +1,13 @@
 ﻿namespace NRedisStack.Search.Literals;
 
-internal class AttributeOptions
+internal class FieldOptions
```

Comment:
I know it doesn't matter as it's internal but I'm not sure about changing a class name (especially when there is not structural change).

You may ignore my comment if you're sure about this.

Reply:
i dont like same thing called as field, attribute and property in the same context. lets get rid of at least one.